capstone/backend/src/main/java/com/google/spanner/CreateSpannerTables.java

```diff
@@ -0,0 +1,157 @@
+                    + "  ItemTypes   ARRAY<STRING(MAX)>"
+                    + ") PRIMARY KEY (UserId, ListId),"
+                    + "  INTERLEAVE IN PARENT Users ON DELETE CASCADE",
+                "CREATE TABLE Stores ("
+                    + "  StoreId      INT64,"
+                    + "  StoreName    STRING(MAX),"
+                    + "  Address      STRING(MAX)"
+                    + ") PRIMARY KEY (StoreId)",
+                "CREATE TABLE Items ("
+                    + "  ItemId            INT64,"
+                    + "  ItemNameAndBrand  STRING(MAX),"
+                    + "  ItemType          STRING(MAX)"
+                    + ") PRIMARY KEY (ItemId)",
+                "CREATE TABLE Inventory ("
+                    + "  StoreId           INT64,"
+                    + "  ItemId            INT64,"
+                    + "  ItemAvailability  STRING(MAX),"
+                    + "  LastUpdated       TIMESTAMP OPTIONS (allow_commit_timestamp=true),"
+                    + "  Price             FLOAT64,"
+                    + "  CONSTRAINT FK_ItemId FOREIGN KEY (ItemId) REFERENCES Items (ItemId)"
+                    + ") PRIMARY KEY (StoreId, ItemId),"
+                    + "INTERLEAVE IN PARENT Stores ON DELETE CASCADE"));
+    try {
+      // Initiate the request which returns an OperationFuture.
+      Database db = op.get();
+      System.out.println("Created database [" + db.getId() + "]");
+    } catch (ExecutionException e) {
+      // If the operation failed during execution, expose the cause.
+      throw (SpannerException) e.getCause();
+    } catch (InterruptedException e) {
+      // Throw when a thread is waiting, sleeping, or otherwise occupied,
+      // and the thread is interrupted, either before or during the activity.
+      throw SpannerExceptionFactory.propagateInterrupt(e);
+    }
+  }
+  
+  static void run(
+      DatabaseClient dbClient,
+      DatabaseAdminClient dbAdminClient,
+      InstanceAdminClient instanceAdminClient,
+      String command,
+      DatabaseId database) {
+    switch (command) {
+      case "createdatabase":
+        createDatabase(dbAdminClient, database);
+        break;
+      default:
+        printUsageAndExit();
+    }
+  }
+  
+  static void printUsageAndExit() {
+    System.err.println("Usage:");
+    System.err.println("    SpannerExample <command> <instance_id> <database_id>");
+    System.err.println("");
+    System.err.println("Examples:");
+    System.err.println("    SpannerExample createdatabase my-instance step39-db");
+    System.exit(1);
+  }
+  
+  public static void main(String[] args) throws Exception {
+    if (args.length != 3) {
+      printUsageAndExit();
+    }
+    // [START init_client]
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    try {
+      String command = args[0];
+      DatabaseId db = DatabaseId.of(options.getProjectId(), args[1], args[2]);
+      // [END init_client]
```

Comment:
delete this. there're a number of `// [START/END init_client]` comments in this file, delete the ones that you don't need.